Added copy and paste

jimrye opened the pull request:
Added copy and paste with a special function for extended BASIC lines.

BGPierce commented:
I will pull and check it out as soon as I can.
Thanks!!


-----Original Message-----
From: snowdogz <notifications@github.com>
To: VCCE/VCC <VCC@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Sent: Mon, Oct 19, 2020 8:14 pm
Subject: [VCCE/VCC] Added copy and paste (#16)

Added copy and paste with a special function for extended BASIC lines.
You can view, comment on, or merge this pull request online at:
  https://github.com/VCCE/VCC/pull/16
Commit Summary
   
   - Added copy and paste

File Changes
   
   -  M Vcc.c (12) 
   -  M Vcc.rc (8) 
   -  M Vcc.sln (32) 
   -  M coco3.cpp (401) 
   -  M coco3.h (3) 
   -  M config.c (4) 
   -  M config.h (1) 
   -  M keyboard.c (12) 
   -  M keyboard.h (6) 
   -  M pakinterface.c (4) 
   -  M resource.h (3) 
   -  M tcc1014graphics.c (3) 
   -  M tcc1014graphics.h (1) 

Patch Links:
   
   - https://github.com/VCCE/VCC/pull/16.patch
   - https://github.com/VCCE/VCC/pull/16.diff
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

jimrye commented:
Cool! I THINK I have the project settings right.


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Bill Pierce<mailto:notifications@github.com>
Sent: Monday, October 19, 2020 11:25 PM
To: VCCE/VCC<mailto:VCC@noreply.github.com>
Cc: snowdogz<mailto:redteck@hotmail.com>; Author<mailto:author@noreply.github.com>
Subject: Re: [VCCE/VCC] Added copy and paste (#16)

I will pull and check it out as soon as I can.
Thanks!!


-----Original Message-----
From: snowdogz <notifications@github.com>
To: VCCE/VCC <VCC@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Sent: Mon, Oct 19, 2020 8:14 pm
Subject: [VCCE/VCC] Added copy and paste (#16)

Added copy and paste with a special function for extended BASIC lines.
You can view, comment on, or merge this pull request online at:
  https://github.com/VCCE/VCC/pull/16
Commit Summary

- Added copy and paste

File Changes

- M Vcc.c (12)
- M Vcc.rc (8)
- M Vcc.sln (32)
- M coco3.cpp (401)
- M coco3.h (3)
- M config.c (4)
- M config.h (1)
- M keyboard.c (12)
- M keyboard.h (6)
- M pakinterface.c (4)
- M resource.h (3)
- M tcc1014graphics.c (3)
- M tcc1014graphics.h (1)

Patch Links:

- https://github.com/VCCE/VCC/pull/16.patch
- https://github.com/VCCE/VCC/pull/16.diff
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/VCCE/VCC/pull/16#issuecomment-712564225>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKDTMSKSQVCYFHPODZHGY73SLT7LDANCNFSM4SXBQX7Q>.

jimrye committed:
PATCH: Added "Paste with New" and fixed "Copy Text"

Added option to "Paste with New" and trimmed the last character from a 32 column "Copy Text" to remove "garbage" caused by flashing cursor.

jimrye committed:
Merge branch 'VCC_2.0.1c' into copy-paste
